.github/bot/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "auto-merge-eip",
+  "version": "1.0.0",
+  "description": "auto merge eip",
+  "main": "index.js",
+  "scripts": {
+    "test": "NODE_ENV=testing jest __tests__",
+    "build": "ncc build ./src/index.ts",
+    "watch": "ncc build ./src/index.ts --watch",
+    "it": "node -r dotenv/config dist/index.js",
+    "prettier-changed": "prettier --write"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@actions/core": "^1.2.6",
+    "@actions/github": "^4.0.0",
+    "@octokit/types": "^6.12.1",
+    "@typescript-eslint/parser": "^4.16.1",
+    "@zeit/ncc": "^0.22.3",
+    "dotenv": "^8.2.0",
+    "eslint": "^7.21.0",
+    "front-matter": "^4.0.2",
+    "node-fetch": "^2.6.1",
+    "typescript": "^4.1.5"
```

Comment:
Several of these should be moved to `devDependencies`.  TypeScript, linters, etc. at the least.